### DISCUSSION TITLE
Can you use a title attribute as the content for tooltips?

### DISCUSSION THREAD

**Discussion by in3rsha** [atomiks/tippyjs#1056] [Q&A]
I have some tooltips that are set up like this:

HTML:
```
<button id="myButton" title="This is a tooltip">My Button</button>
```

Javascript:

```
tippy('#myButton', {
  content: "Some stuff",
});
```

Is there any way I can set the `content:` to use the contents of the `title=` attribute as the contents of the tooltip?

My reason is that like to use the `title=` attribute as a fallback for when javascript is disabled. So if javascript is disabled, there will still be some form of tooltip available.

I'm guessing I'd also need to remove the `title=` attribute from the element at the same time to prevent two tooltips (the default and Tippy.js) from loading at the same time.

Is this possible?

**Comment by melvincwng** [comment]
@in3rsha, I don't think it's possible to use both 'title' and 'content' at the same time.

I've tested it out on my end, and it results in two tooltips showing (as you've also already mentioned).

For consistency purposes, my recommendation would be to just use 'content' as per the library's documentation.

Hope this clarifies.
